src/components/IncidentList.vue

```diff
@@ -1,24 +1,52 @@
 <template>
   <div :class="$style.wrapper">
     <center-wrapper>
+      <incident-filter v-model="selectedCity" :options="allCities" />
+
+      <div :class="$style['filter-description']">
```

Comment:
If you want, you can create a seperate component for the results

Reply:
Feels like overkill to me, I'll skip this